### ISSUE TITLE
Unresponsive Shake Message

### ISSUE THREAD

**Issue by Davnik** [HeavensGate/Eternal#76]
"You shake [mob], but they do not respond... Maybe they have S.S.D?"
Please remove "Maybe they have S.S.D?"
The message should read "You shake [mob], but they do not respond..."

**Comment by Davnik** [comment]
Jakk, what were your recommendations as to an alternative?

**Comment by Jacquar** [comment]
Apologies for my long absence from the github issues.

"You shake [mob] lightly, but they remain unresponsive. If they're suffering SSD, perhaps they should be taken to a place of safety?"

Too wordy? I want to imply that SSDs should be respected by crew, upon policy/law.

Too wordy? It's vague, so

**Comment by Mewchild** [comment]
@Davnik @Jacquar What's the status on this? SSD is a commonly accepted IC term

**Comment by Davnik** [comment]
I believe Jakk's option is too wordy. I believe my option might not be preferable, as it removes the meta-suggestion that they probably have SSD. At the bare minimum, I recommend not placing a capital after this ellipsis and removing the dots in SSD, as they make only a shoddy partial abbreviation. The grammar and presumptuousness here kill me.

**Comment by Jacquar** [comment]
"You check *mob*. Alive, but unresponsive."

Avoids the he/she/they/it issue, avoids any unnecessary and potentially
contradictory information on health or physical traits that might
contradict a given species, doesn't quote SSD, simple as possible.
Downsides?

**Comment by Davnik** [comment]
- "Alive, but unresponsive" sounds best, for a comatose person.
- I agree, directly referencing SSD is... awkward, maybe wrong. 
- Downside: Staff have shown interest in mentioning the individual should be taken to cryosleep.

**Comment by Davnik** [comment]
"You check *mob*. Alive, but unresponsive." is what we're going with. 

Discuss/uncertain is now resolved. As discussions have pointed out, we should not imply it is a player's responsibility to deal with an SSD; staff will sweep those along in due time, or players will do so of IC volition. Snore logs will be dealt with by some other issue, and it all comes full circle. Other options have come off as too formal, too much of a fourth-wall inner voice guiding you to do something arbitrarily, etc. Short, simple, to the point, make what you will of the faceplantee.

**Comment by killer99603** [comment]
As long as no one else has done it yet, I got dis.
